projects/plugins/jetpack/class.json-api.php

```diff
@@ -205,19 +205,20 @@ public function add( WPCOM_JSON_API_Endpoint $endpoint ) {
 	 * Determine if a string is truthy. If it's not a string, which can happen with
 	 * not well-formed data coming from Jetpack sites, we still consider it a truthy value.
 	 *
-	 * @param string $value "1", "t", and "true" (case insensitive) are truthy, everything else isn't.
+	 * @param mixed $value 1, "1", "t", and "true" (case insensitive) are truthy, everything else isn't.
 	 * @return bool
 	 */
 	public static function is_truthy( $value ) {
 		if ( true === $value ) {
 			return true;
 		}
 
-		if ( ! is_string( $value ) ) {
+		if ( ! is_string( $value ) && ! is_int( $value ) ) {
 			return false;
 		}
 
 		switch ( strtolower( (string) $value ) ) {
+			case 1:
```

Comment:
Should never hit this, since `strtolower()` will always output a string.

Reply:
Great catch, thank you! I've replaced the `is_int` check and the added `case` with an explicit check for the only allowed value in both methods, since `switch` is doing a loose comparison anyway.